great_expectations/core/batch.py

```diff
@@ -594,17 +598,28 @@ def __init__(
 
     @staticmethod
     def _validate_runtime_batch_request_specific_init_parameters(
-        runtime_parameters: dict,
-        batch_identifiers: dict,
+        runtime_parameters: Optional[dict],
+        batch_identifiers: Optional[dict],
         batch_spec_passthrough: Optional[dict] = None,
     ) -> None:
-        if not (runtime_parameters and (isinstance(runtime_parameters, dict))):
+        # we either have both runtime_parameters and batch_identifiers, or neither
+        if not (
+            (not runtime_parameters and not batch_identifiers)
+            or (runtime_parameters and batch_identifiers)
+        ):
+            raise ValueError(
```

Comment:
@donaldheppner Just a suggestion: the operand of the if clause can be simplified to the exclusive `OR` (you have the negation of the exclusive `NOR`).  Please feel to apply or not as you deem appropriate.  Thanks.

Reply:
I'll consider this going forward